packages/api/tests/lambdas/test-index-from-database.js

```diff
@@ -0,0 +1,152 @@
+process.env.stackName = randomString();
+
+process.env.ExecutionsTable = randomString();
+process.env.CollectionsTable = randomString();
+process.env.GranulesTable = randomString();
+process.env.PdrsTable = randomString();
+process.env.ProvidersTable = randomString();
+process.env.RulesTable = randomString();
+
+const tables = {
+  collectionsTable: process.env.CollectionsTable,
+  executionsTable: process.env.ExecutionsTable,
+  granulesTable: process.env.GranulesTable,
+  pdrsTable: process.env.PdrsTable,
+  providersTable: process.env.ProvidersTable,
+  rulesTable: process.env.RulesTable
+};
+
+const executionModel = new models.Execution();
+const collectionModel = new models.Collection();
+const granuleModel = new models.Granule();
+const pdrModel = new models.Pdr();
+const providersModel = new models.Provider();
+const rulesModel = new models.Rule();
+
+async function addFakeData(numItems, factory, model, factoryParams = {}) {
+  const items = [];
+
+  /* eslint-disable no-await-in-loop */
+  for (let i = 0; i < numItems; i += 1) {
+    const item = factory(factoryParams);
+    items.push(item);
+    await model.create(item);
+  }
+  /* eslint-enable no-await-in-loop */
+
+  return items;
+}
+
+function searchEs(type) {
+  const executionQuery = new Search({}, type, esIndex);
+  return executionQuery.query();
+}
+
+test.before(async () => {
+  esIndex = randomString();
+  // create esClient
+  esClient = await Search.es('fakehost');
+
+  // add fake elasticsearch index
+  await bootstrap.bootstrapElasticSearch('fakehost', esIndex);
+
+  await aws.s3().createBucket({ Bucket: process.env.system_bucket }).promise();
+
+  await executionModel.createTable();
+  await collectionModel.createTable();
+  await granuleModel.createTable();
+  await pdrModel.createTable();
+  await providersModel.createTable();
+  await rulesModel.createTable();
+
+  // upload workflow lists
+  const workflowsListKey = `${process.env.stackName}/workflows/list.json`;
+  await aws.promiseS3Upload({
+    Bucket: process.env.system_bucket,
+    Key: workflowsListKey,
+    Body: JSON.stringify(workflowList)
+  });
+
+  const workflow = `${process.env.stackName}/workflows/${workflowList[0].name}.json`;
+  await aws.promiseS3Upload({
+    Bucket: process.env.system_bucket,
+    Key: workflow,
+    Body: JSON.stringify(workflowList[0])
+  });
+});
+
+test.after.always(async () => {
+  await esClient.indices.delete({ index: esIndex });
+
+  await executionModel.deleteTable();
+  await collectionModel.deleteTable();
+  await granuleModel.deleteTable();
+  await pdrModel.deleteTable();
+  await providersModel.deleteTable();
+  await rulesModel.deleteTable();
+
+  await aws.recursivelyDeleteS3Bucket(process.env.system_bucket);
+});
+
+test('No error is thrown if nothing is in the database', async (t) => {
+  t.notThrows(async () => indexFromDatabase.indexFromDatabase(esIndex, tables));
+});
+
+test.only('index executions', async (t) => {
```

Comment:
The `only` is still on. Did you mean this?

Reply:
definitely not!! this is why PR reviews are so great! thank you!